ui/raidboss/data/05-shb/raid/e10n.js

```diff
@@ -41,12 +42,7 @@ export default {
       netRegexJa: NetRegexes.startsUsing({ source: '影の王', id: '56B7', capture: false }),
       alertText: (data, _, output) => output.text(),
       outputStrings: {
-        text: {
-          en: 'Go Front',
-          de: 'Geh nach vorn',
-          ja: '前へ',
-          cn: '去前面',
-        },
+        text: Outputs.goFront,
```

Comment:
Thanks for this! I was a little surprised this hadn't come up in any content recent enough to be useful, and I didn't have the time to do it myself when submitting this.